### DISCUSSION TITLE
Feature request: Save Descriptor to SD card (and recover it from SD)

### DISCUSSION THREAD

**Discussion by odudex** [selfcustody/krux#262] [Ideas]
Moving issues and feature requests from odudex repo to here, this is from [Bitconheiro](https://github.com/bitcoinheiro):

After importing a multisig descriptor (or any descriptor) via QR would be great if there would be an option to save it to the microSD in a standard format for importing to other devices/coordinators and/or back to krux when/if needed.

This way I don't need to scan it from the coordinator every time I want to check an address of a multisig wallet.

Original: https://github.com/odudex/krux/issues/11

**Comment by tadeubas** [comment]
Wow, nice coincidence! I am already doing this feature!
